src/pages/members.js

```diff
@@ -0,0 +1,137 @@
+import NavBar from "../components/navBar";
+import Dropdown from "../assets/chevron_down.svg";
+
+import { useEffect, useState } from "react";
+import { GET } from "../utils/fetch";
+import { Link } from "react-router-dom";
+
+const getUsers = async () => {
+  return await GET("users");
+};
+
+function Members() {
+  const [users, setUsers] = useState([]);
+  const [searchInput, setSearchInput] = useState("");
+  const [sortsToggle, setSortsToggle] = useState(false);
+  const [sortText, setSortText] = useState("Name (A-Z)");
+
+  useEffect(() => {
+    getUsers().then((response) => {
+      const sortedUsers = response.sort((a, b) => a.name.localeCompare(b.name));
+      setUsers(sortedUsers);
+    });
+  }, []);
+
+  const handleSearch = (e) => {
+    setSearchInput(e.target.value.trim());
+  };
+
+  const filteredUsers = users.filter((user) =>
+    user.name.toLowerCase().includes(searchInput.toLowerCase())
+  );
+
+  const switchOrder = (e) => {
+    e.preventDefault();
+
+    var order = e.target.value;
+    setSortText(order);
+
+    var ordered = users;
+    switch (order) {
+      case "Name (A-Z)":
+        ordered = ordered.sort((a, b) => a.name.localeCompare(b.name));
+        break;
+      case "Name (Z-A)":
+        ordered = ordered.sort((a, b) => b.name.localeCompare(a.name));
+        break;
+      default:
+        ordered = users;
+        break;
+    }
+    setUsers(ordered);
+  };
+
+  const toggleSorts = () => {
+    setSortsToggle((prevState) => !prevState);
```

Comment:
`setSortsToggle(!prevState);` is enough

Reply:
I don't think so :sweat_smile: 
![image](https://github.com/Subvisual-Academy/space/assets/61982705/071df9e4-4722-4703-bdbe-95e2fe2b0c7e)

Reply:
It should be `setSortsToggle(!sortsToggle)` instead of `setSortsToggle(!prevState)`